### PR TITLE
week-01

### PR DESCRIPTION
**Pull request by RokHoang**

### CODE REVIEW

File: assignments/week-01/eval.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+func trim(text string) string {
+	space := regexp.MustCompile(`\s+`)
+	return space.ReplaceAllString(text, " ")
+}
+
+func divide(n1 int, n2 int) (float64, error){
+	if (n2 == 0){
+		return 0.0,errors.New("Cannot be divided by 0 (zero)")
```
Comment (hoksilato) [inline assignments/week-01/eval.go]: Return `0` is good enough.

File: assignments/week-01/eval.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	if (len(s)<3){
+		return errors.New("Too few arguments")
+	}
+
+	if (len(s)>3){
+		return errors.New("Too many arguments")
+	}
+
+	t1,op,t2 := s[0], s[1], s[2]
+	n1, errN1 := strconv.Atoi(t1)
```
Comment (hoksilato) [inline assignments/week-01/eval.go]: You should return if `errN1` is not `nil` so that we don't need to run line 23.

**Review by hoksilato — COMMENTED** [review]
Good work, thank you 👍

**Review by RokHoang — COMMENTED** [review]
week-01

File: assignments/week-01/eval.go (hunk 2 above)
Comment (RokHoang) [reply assignments/week-01/eval.go]: thanks

**Pull request merged**